### DISCUSSION TITLE
Warnings when building Rust Bindings to my Grammar

### DISCUSSION THREAD

**Discussion by chappertron** [tree-sitter/tree-sitter#3670] [Q&A] [answered]
I've been getting the following warnings when building the rust bindings to a grammar I've written

``` shell 
cargo build 
```

```
warning: tree-sitter-lammps@0.0.2: src/parser.c:8299:3: warning: unused label 'next_state' [-Wunused-label]
warning: tree-sitter-lammps@0.0.2:  8299 |   START_LEXER();
warning: tree-sitter-lammps@0.0.2:       |   ^~~~~~~~~~~~~
warning: tree-sitter-lammps@0.0.2: src/tree_sitter/parser.h:169:3: note: expanded from macro 'START_LEXER'
warning: tree-sitter-lammps@0.0.2:   169 |   next_state:                   \
warning: tree-sitter-lammps@0.0.2:       |   ^~~~~~~~~~
warning: tree-sitter-lammps@0.0.2: 1 warning generated.
```

These have only started showing up after drastically simplifying the grammar. 

Does anybody have an idea what is causing these warnings? Is this caused by an unused rule or something? 

Thank you in advance for any help!

Extra information:
CLI Version: 0.23.0
Rust version:  rustc 1.81.0 
I'm not sure which compiler is being used to compile the bindings, but likely Apple clang.
EDIT: I've not had the same issue compiling `tree-sitter-bash` or `tree-sitter-python` rust bindings.

**Comment by amaanq** [comment]
it's innocuous, you can ignore.

**Reply by chappertron** [reply]
Thank you, although it would be nice to not have this. I'll look into ways to suppress the warnings.

**Reply by amaanq** [reply, marked as answer]
`.flag_if_supported("-Wno-unused-label")` in your build.rs on the `c_config` variable.

**Reply by chappertron** [reply]
Thank you!
